src/tests/functional/navigation_tests.ts

```diff
@@ -115,6 +115,24 @@ export class NavigationTests extends TurboDriveTestCase {
     this.assert.equal(await this.pathname, "/src/tests/fixtures/one.html")
     this.assert.equal(await this.visitAction, "restore")
   }
+
+  async "test lazy loading within a details element"() {
+    await this.nextBeat
+
+    const frameContents = "#loading-lazy turbo-frame h2"
+    this.assert.notOk(await this.hasSelector(frameContents))
+
+    await this.clickSelector("#loading-lazy summary")
+    await this.nextBeat
```

Comment:
@sstephenson this test is currently failing. I suspect the "next beat" after the button click waits for the `<details>` element to expand, instead of waiting for the `<turbo-frame>` contents to get loaded.

Is that a correct assumption? When I test this out locally, the frame lazily loads as expected. I think the test failure has to do with the test harness and assertions.